Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,323 @@
+    @State private var model = Model()
+    
+    /// A Boolean value indicating whether the settings sheet is presented.
+    @State private var isPresented = false
+    
+    var body: some View {
+        SceneView(
+            scene: model.scene,
+            graphicsOverlays: [model.graphicsOverlay],
+            analysisOverlays: [model.analysisOverlay]
+        )
+        .onAppear {
+            model.setupAnimation()
+        }
+        .onDisappear {
+            model.stopAnimating()
+        }
+        .overlay(alignment: .top) {
+            HStack {
+                Text("Visibility:")
+                Text(model.targetVisibility.label)
+            }
+            .frame(maxWidth: .infinity)
+            .padding(.vertical, 6)
+            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button("Settings") {
+                    isPresented = true
+                }
+                .sheet(isPresented: $isPresented) {
+                    settingsSheet
+                }
+            }
+        }
+    }
+    
+    /// The settings configuration sheet for adjusting the observer's height.
+    private var settingsSheet: some View {
+        NavigationStack {
+            Form {
+                let heightRange = 20.0...70.0
+                LabeledContent(
+                    "Observer Height",
+                    value: Measurement(value: model.height, unit: UnitLength.meters).formatted()
+                )
+                Slider(
+                    value: $model.height,
+                    in: heightRange,
+                    step: 1
+                ) {
+                    Text("Observer Height")
+                } minimumValueLabel: {
+                    Text(Measurement(value: heightRange.lowerBound, unit: UnitLength.meters).formatted())
+                } maximumValueLabel: {
+                    Text(Measurement(value: heightRange.upperBound, unit: UnitLength.meters).formatted())
+                }
+                .listRowSeparator(.hidden, edges: .top)
+            }
+            .presentationDetents([.fraction(0.25)])
+            .navigationTitle("Settings")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        isPresented = false
+                    }
+                }
+            }
+        }
+    }
+}
+
+private extension ShowLineOfSightBetweenGeoelementsView {
+    /// View model responsible for setting up the 3D scene, graphics, animation,
+    /// and performing line of sight analysis between moving and static geoelements.
+    @MainActor
+    @Observable
+    final class Model {
+        /// A set of predefined waypoints for animating the taxi's movement.
+        @ObservationIgnored private let points = [
```

Comment:
```suggestion
        private let points = [
```
only variables would be annotated by the `@Observable` macro.